maquetado/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="book-detail">
+    <p class="book-title">Titulo del Libro</p>
+    <p class="book-description book-genre">(género)</p>
```

Comment:
Mmmm why using 2 classes for this instead of just 1 ? I would leave `book-genre`